scripts/buildAndTest.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+if ! type autopep8 > /dev/null; then
+  echo "'autopep8' required for build."
+  exit 1;
+fi
+
 PEPLINES=$(autopep8 --recursive ./src/ --diff | wc -l)
```

Comment:
so with my clowny code beforehand, what did this line do? since autopep8 was missing, it probably just wrote some output to stderr and then the counted lines of stdout was 0, so it skipped?

wow thats lucky :O